src/frontend/src/optimizer/plan_node/stream_table_scan.rs

```diff
@@ -199,7 +200,10 @@ impl StreamTableScan {
             })),
             stream_key,
             operator_id: self.base.id.0 as u64,
-            identity: format!("{}", self),
+            identity: {
+                let s = format!("{}", self);
+                s.replace("StreamTableScan", "Chain")
```

Comment:
Ahhh 😅 I prefer to leave it as is

Reply:
maybe It should be same as the dashboard

Reply:
So how about 

1. rename Chain to StreamTableScan in dashboard
2. or rename StreamTableScan to Chain in operator? 

I prefer 1